### ISSUE TITLE
Podocyte hypertrophy

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4816]
**Preferred term label:**
Podocyte hypertrophy

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://www.ncbi.nlm.nih.gov/pubmed/16120818

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0031265

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This article was from an experimental study on rats. This term is used in the clinical literature as well but I have not found a good definition.
